### PR TITLE
Feature/pelagos 3301 load geometry on hover

### PR DESCRIPTION
**Pull request by snguyen1**
- Include geometry in from the search on server-side
- Adding geometry hovering event
- Disable Show All Extent

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js [hunk 1]
```diff
@@ -167,6 +167,24 @@ function addRows() {
         var row = document.createElement("tr");
         $(row).attr("udi", data["udi"]);
         $(row).attr("datasetid", dataset["_id"]);
+        if (data["geometry"]) {
+            $(row).hover(function () {
+
+                if (!$('#show_all_extents_checkbox').is(':checked')) {
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js]: Double quotes on line 173, 175, 180, 182 etc

File: src/Pelagos/Bundle/AppBundle/Controller/UI/DataDiscoveryController.php [hunk 2]
```diff
@@ -263,10 +263,9 @@ public function searchAction(Request $request)
 
         if ((count($results) - $currentIndex) >= $bulkSize) {
             $results = array_slice($results, $currentIndex, $bulkSize);
-        }
-        else {
+        } else {
             //return the last bulk of data (the last bulk is less than bulk size)
-            $results = array_slice($results, $currentIndex, count($results) - $currentIndex);
+            $results = array_slice($results, $currentIndex, (count($results) - $currentIndex));
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Controller/UI/DataDiscoveryController.php]: Does this change the number of results?

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js [hunk 3]
```diff
@@ -360,9 +376,9 @@ function showDatasets(by,id) {
                     loadData(by, id);
                 }
             });
-            if (myGeoViz.getFilter()) {
+            if (geo_filter) {
                 $("#clearGeoFilterButton").button("enable");
-            }
+             }
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js]: Remove space.

File: src/Pelagos/Bundle/AppBundle/Controller/UI/DataDiscoveryController.php (hunk 2 above)
Comment (snguyen1) [reply src/Pelagos/Bundle/AppBundle/Controller/UI/DataDiscoveryController.php]: no but phpcs didn't like it

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js (hunk 3 above)
Comment (snguyen1) [reply src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js]: I'll fix it in feature/PELAGOS-3319 (the last subtask)

**Pull request merged**